docs/assignment1.md

```diff
@@ -1 +1,53 @@
-Reverted this file
+## Introduction									
+Author(s): `Noé Rousset`
+
+This document introduces the Fantasy Soccer Project, addressing the specifications and requirements of the system and the libraries to be used to complete the project.
+ 
+This project closely follows the standard format of fantasy football games, in that a user assembles imaginary teams, whose performance is statistically evaluated. Unlike the original format, however, for this game, the data, as well as the schedule, are both provided by the player. 
+With the players added by the user, they can form a team of 11 players consisting of 1 goalkeeper, 4 defenders, 4 midfielders and 2 forwards. The score of each player is also added by the user, and the score of a team is the cumulative score of all the players. 
+After that team is formed, a schedule is set by the user and a tournament is played with other random teams. The teams play 1 on 1 matches in a single-elimination format[1], where the team with the highest total score wins. After a tournament is over, the user can form a new team, start a tournament again with the current team  or close the game.
+Possible improvements to this would be to use data for the players found online instead of the user adding it manually, and changing the way matches are played. In this upgraded version, each player would have the probability of committing a certain action, or an event happening based on their stats. For example a player with a strong attack would have a higher probability of scoring a goal. The score of a team would thus change between each game depending on their performance, with different actions [2]  earning a certain amount of points. 
+
+[1] https://en.wikipedia.org/wiki/Single-elimination_tournament
+
+[2] https://en.wikipedia.org/wiki/Fantasy_football_(association)#Points_scoring
+
+## Features
+
+### Functional features
+Author(s): `Teona Moga`
+
+The functional requirements below represent our understanding of the technical problem to be solved. We defined the constraints that the system needs to adhere to as well as the expectations for its anticipated behaviour.
+
+| ID  | Short name  | Description  |
+|---|---|---|
+| F1  | User-provided data | Allow the user to manually provide player data via the user interface in CSV format. Each row should only contain two values, i.e. name and score, e.g. `Ben Foster,51`. The file shouldn’t have a header row. |
+| F2  | Erroneous data  | In the event user input cannot be processed, present an error detailing the issue to the user.|
```

Comment:
The name of this feature should be more precise (i.e Error reporting), make sure to include a security QR in relation to how much information do you want an error to reveal (Golden rule, we do not report verbose backend errors to the user)